db/migrate/20160621151100_polymorphic_mentions.rb

```diff
@@ -0,0 +1,12 @@
+class PolymorphicMentions < ActiveRecord::Migration
+  def change
+    remove_index :mentions, %i(post_id)
+    remove_index :mentions, %i(person_id post_id)
+    rename_column :mentions, :post_id, :mentions_container_id
+    add_column :mentions, :mentions_container_type, :string, default: "Post", null: false
+    add_index :mentions, %i(mentions_container_id mentions_container_type), name: "index_mentions_on_mc_id_and_mc_type"
```

Comment:
When I try the same, I get the error: MariaDB 10.0.23

But I can create the index manually in the DB.

Anyway, why don't you just add the `length` parameter as in other migrations where indexes are added? WIth the `length` it works.